tests/features/media/image.feature

```diff
@@ -16,7 +16,22 @@ Feature: Image media assets
     Then I should be visiting a media entity
     And I should see "Foobaz"
 
-  @image @javascript @b23435a5
+  @13eacffd
+  Scenario: Cropping should be allowed when creating an image
```

Comment:
It'd be really difficult to test the actual cropping, and that's squarely in Image Widget Crop's wheelhouse anyway. I figure it's good enough to ensure that the crop widget is showing up where and when we expect it to.